### ISSUE TITLE
Install-Module prompting for untrusted repo when *all* repos are trusted

### ISSUE THREAD

**Issue by rkeithhill** [PowerShell/PowerShellGetv2#353]
Running this on Windows 10 WSL w/Ubuntu 18.04 LTS running PS Core 6.1.0

Steps to reproduce
------------------

```powershell
PS /mnt/c/Users/hillr/Git> install-module posh-git -Scope CurrentUser -AllowPrerelease

Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its
InstallationPolicy value by running the Set-PSRepository cmdlet. Are you sure you want to install the modules from
'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): Y

PS /mnt/c/Users/hillr/Git> Get-PSRepository

Name                      InstallationPolicy   SourceLocation
----                      ------------------   --------------
powershell-keysight       Trusted              https://artifactory.it.keysight.com/artifactory/api/nuget/powershell-...
PSGallery                 Trusted              https://www.powershellgallery.com/api/v2

```

Expected behavior
-----------------

I should not be prompted to install from a trusted the repo.

Actual behavior
---------------

I am prompted to install from a repo that is **already** trusted.

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.1.0
PSEdition                      Core
GitCommitId                    6.1.0
OS                             Linux 4.4.0-17763-Microsoft #55-Microsoft Sat Oct 06 18:05:00 PST 2018
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

```powershell
> Get-Module -ListAvailable PowerShellGet,PackageManagement
ModuleType Version    Name                                PSEdition ExportedCommands
---------- -------    ----                                --------- ----------------
Script     1.1.7.2    PackageManagement                   Desk      {Find-Package, Get-Package, Get-PackageProvider,...
Script     1.6.7      PowerShellGet                       Desk      {Find-Command, Find-DSCResource, Find-Module, Fi...
```

```powershell
> Get-PackageProvider -ListAvailable
Name                     Version          DynamicOptions
----                     -------          --------------
NuGet                    2.8.5.210        Destination, ExcludeVersion, Scope, SkipDependencies, Headers, FilterOnTag...
PowerShellGet            1.6.7.0          PackageManagementProvider, Type, Scope, AllowClobber, SkipPublisherCheck, ...
```

**Comment by edyoung** [comment]
Notes: I can repro this, but only in WSL (pwsh on Windows doesn't show this, nor does it repro on regular Ubuntu). I was also able to repro with only one repository registered, so doesn't seem to be related to having 2. Doesn't repro with InputObject parameterset instead of Name parameterset.

**Comment by edyoung** [comment]
Fixed by https://github.com/OneGet/oneget/pull/451

**Comment by edyoung** [comment]
fixed in 2.1.4/packagemanagement 1.4
